kamu-core/src/domain/repos/metadata_chain.rs

```diff
@@ -59,6 +61,14 @@ pub trait MetadataChain: Send + Sync {
         opts: AppendOpts<'a>,
     ) -> Result<Multihash, AppendError>;
 
+    /// Loads the block from bytes and appends it to the chain
+    async fn append_block_from_bytes<'a>(
+        &'a self,
+        hash: &Multihash,
+        block_bytes: Bytes,
```

Comment:
What is the advantage of passing `Bytes` here instead of a slice `&[u8]`?

Reply:
I did not invent this. This extends the existing code path in `get_block` about deserializing block content from disk